### PR TITLE
docs: add markAllAsTouched and clear method to doc

### LINKED ISSUE #30439: Add markAllAsTouched and clear method to example doc
# 📚 Docs or angular.io bug report

### Description

 A clear and concise description of the problem...

FormArray has 2 new methods markAllAsTouched and clear add both to example to showcase the functionality.

### What's the affected URL?**
 https://angular.io/...

## 🔥 Exception or Error
<pre><code>

</code></pre>

### PR DESCRIPTION
**Pull request by santoshyadavdev**
Fixes #30439

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #30439

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/reactive-forms.md [hunk 1]
```diff
@@ -460,6 +472,23 @@ Initially, the form contains one `Alias` field. To add another field, click the
 
 </div>
 
+### Step 5: Removing Controls from FormArray
+
+Once controls are added dynamically it also possible to remove the controls dynamically from the array using `removeAt()` function, which takes the index position and removes the control.
```
Comment (Splaktar) [inline aio/content/guide/reactive-forms.md]:
```suggestion
Once controls are added to a `FormArray` dynamically, it is possible to remove them from the array using the `removeAt()` function. The function removes the control at the specified index.
```

File: aio/content/guide/reactive-forms.md [hunk 2]
```diff
@@ -460,6 +472,23 @@ Initially, the form contains one `Alias` field. To add another field, click the
...
+
+<code-example path="reactive-forms/src/app/profile-editor/profile-editor.component.ts" region="remove-alias" header="src/app/profile-editor/profile-editor.component.ts (remove alias)">
+
+</code-example>
+
+### Step 6: Clear All Controls from FormArray
+
+It is possible to clear all controls from FormArray. You can use `clear()` function.
```
Comment (Splaktar) [inline aio/content/guide/reactive-forms.md]:
```suggestion
It is possible to clear all controls from a FormArray. Call the `clear()` function to accomplish this.
```

File: aio/content/guide/reactive-forms.md [hunk 3]
```diff
@@ -366,6 +366,18 @@ HTML5 has a set of built-in attributes that you can use for native validation, i
 
 </div>
 
+### Step 3: Marking all Form field as touched
+
+It is possible to mark all fields of Form using `markAllAsTouched()`. This is really useful in case you have multiple forms on your view and want to display validation message for both on a single button click.
```
[the remark this answers is not in the document]
Comment (santoshyadav198613) [reply aio/content/guide/reactive-forms.md]: Done.

File: aio/content/guide/reactive-forms.md (hunk 1 above)
Comment (santoshyadav198613) [reply aio/content/guide/reactive-forms.md]: Done.

File: aio/content/guide/reactive-forms.md (hunk 2 above)
Comment (santoshyadav198613) [reply aio/content/guide/reactive-forms.md]: Done.

**Comment by brandonroberts** [comment]
I'm not sure these additions need to go here. They don't provide much in the context of the guide.

**Comment by santoshyadav198613** [comment]
> I'm not sure these additions need to go here. They don't provide much in the context of the guide.

Hi @brandonroberts ,
I think steps will help developers on how to remove the controls added as we are already showing that in doc, we should show how to clear or remove. Let me know your thoughts, I will be happy to make changes.

**Comment by AndrewKushnir** [comment]
Hi @santoshyadavdev, thanks for creating this PR and sorry for the delay.

I agree with Brandon, given that the current guide is quite long already, we should avoid adding more information there and may be look at possible ways to simplify/restructure it (which is outside of the context of this PR).

My proposal is to look at the individual docs of the mentioned API functions (such as `markAllAsTouched`) and extend their descriptions as needed to provide more information to users.

Thank you.

**Pull request closed without merging**